centreon/www/include/configuration/configGenerate/xml/moveFiles.php

```diff
@@ -348,15 +348,21 @@
                     }
                 }
             } else {
-                passthru("echo 'SENDCFGFILE:" . $host['id'] . "' >> $centcore_pipe", $return);
+                passthru(
+                    escapeshellcmd("echo 'SENDCFGFILE:{$host['id']}'") . ' >> ' . escapeshellcmd($centcore_pipe),
```

Comment:
You can use `escapeshellarg` when you have only one argument.
```suggestion
                    escapeshellcmd("echo 'SENDCFGFILE:{$host['id']}'") . ' >> ' . escapeshellarg($centcore_pipe),
```